gpt_index/indices/query/query_runner.py

```diff
@@ -168,45 +169,13 @@ def _get_query_obj(
 
         return query_obj
 
-    def query(
+    def query_transformed(
```

Comment:
i spent like 10 mins thinking about a better way to do this and can't immediately think of one 

we are, however, going to need an async version of this.

Reply:
Addressed by https://github.com/jerryjliu/llama_index/pull/929